catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -176,8 +217,8 @@ private boolean hasMatch(Metacard local, List<Metacard> remoteMetacards) {
     }
 
     private RemoteRegistryResults getRemoteRegistryMetacardsMap() throws FederationAdminException {
-        Map<String, Metacard> remoteRegistryMetacards = new HashMap<>();
-        List<String> failedQueries = new ArrayList<>();
+        Map<String, Metacard> remoteRegistryMetacards = new ConcurrentHashMap<>();
```

Comment:
The concurrent and synchronized collections were a leftover from how i was trying to do it initially.  Will make the RemoteRegistryResults collections immutable.